util/fetch_document_format_codelist.py

```diff
@@ -0,0 +1,59 @@
+"""
+Updates the document format codelist from the IANA media types CSV files.
+"""
+
+import csv
+import os
+import requests
+
+from helper import schema_dir
+
+
+def get_and_parse_csv(url):
+    response = requests.get(url)
+    response.raise_for_status()
+    content = response.content.decode('utf-8')
+    return csv.DictReader(content.splitlines(), delimiter=',')
+
+
+if __name__ == '__main__':
+    # https://www.iana.org/assignments/media-types lists CSV files for each registry
+    registries = [
+        'application',
+        'audio',
+        'font',
+        'image',
+        'message',
+        'model',
+        'multipart',
+        'text',
+        'video'
+    ]
+
+    codes = {}
+
+    for registry in registries:
+        csvreader = get_and_parse_csv(f'https://www.iana.org/assignments/media-types/{registry}.csv')
+        next(csvreader)
+        for row in csvreader:
+            code = row['Template']
+            title = row['Name']
+            # If the type has no value in the template column, construct a code from the registry and name
+            if code == "":
+                code = f'{registry}/{title}'
```

Comment:
Let's use Python's `logging` to log a warning.

Reply:
Done in https://github.com/open-contracting/standard/pull/1308/commits/bf809b43536493308a45c3dba75c6d432ceaead4. Should we also log the removal of deprecated and obsolete codes?

Reply:
Good idea!

Reply:
Done in https://github.com/open-contracting/standard/pull/1314/commits/1d9a59f60e03b6bd9e0290e147437d647973b5b1